src/core/main.c

```diff
@@ -141,6 +147,13 @@ void lvgl_init() {
 int main(int argc, char *argv[]) {
     pthread_mutex_init(&lvgl_mutex, NULL);
 
+#ifdef EMULATOR_BUILD
+    global_sdl_mutex = SDL_CreateMutex();
+    if (global_sdl_mutex == NULL) {
+        // Handle error: SDL_CreateMutex failed
```

Comment:
Please add a log entry.  Good to know if it ever fails.

Reply:
done